merge_streams_block.py

```diff
@@ -0,0 +1,54 @@
+from nio.common.block.attribute import Input
+from nio.common.block.base import Block
+from nio.common.discovery import Discoverable, DiscoverableType
+from nio.common.signal.base import Signal
+from nio.metadata.properties import VersionProperty, TimeDeltaProperty, \
+    BoolProperty
+from nio.modules.scheduler import Job
+
+
+@Input('input_2')
+@Input('input_1')
+@Discoverable(DiscoverableType.block)
+class MergeStreams(Block):
+
+    """ Take two input streams and combine signals together. """
+
+    expiration = TimeDeltaProperty(default={})
+    notify_once = BoolProperty(default=True)
+    version = VersionProperty('0.1.0')
+
+    def __init__(self):
+        super().__init__()
+        self._signals = { "input_1": {}, "input_2": {} }
+        self._expiration_jobs = { "input_1": None, "input_2": None }
+
+    def process_signals(self, signals, input_id='default'):
```

Comment:
Since we're editing stuff, let's change the default from `default` to `input_1`. If `default` ever came through bad things would happen